### PR TITLE
Fixed upload scripts to support globs

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,23 +47,25 @@ MinkDebugExtension comes with three tiny, but powerful scripts:
...
 ```
 
 #### Uploading screenshots
   
-`vendor/lakion/mink-debug-extension/travis/tools/upload-screenshots <glob_path>` - uploades images to [imgur.com](http://imgur.com) and returns a list of file names and urls to them.
-  
-Requires Imgur API key to be set as environmental varialbe `$IMGUR_API_KEY`. Usage:
+`vendor/lakion/mink-debug-extension/travis/tools/upload-screenshots "<glob_path>"` - uploades images to [imgur.com](http://imgur.com) and returns a list of file names and urls to them. 
+
+Glob paths must be quoted to work correctly. Requires Imgur API key to be set as environmental varialbe `$IMGUR_API_KEY`. Usage:
```
Comment (lchrusciel) [inline README.md]: _Glob_?

Comment (pamil) [inline README.md]: Glob :)

Comment (pamil) [inline README.md]: https://en.wikipedia.org/wiki/Glob_(programming)

File: README.md [hunk 2]
```diff
@@ -47,23 +47,25 @@ MinkDebugExtension comes with three tiny, but powerful scripts:
...
+$ vendor/lakion/mink-debug-extension/travis/tools/upload-textfiles "logs/*.log"
 $ vendor/lakion/mink-debug-extension/travis/tools/upload-textfiles README.md
 ```
 
 #### Uploading screenshots
   
-`vendor/lakion/mink-debug-extension/travis/tools/upload-screenshots <glob_path>` - uploades images to [imgur.com](http://imgur.com) and returns a list of file names and urls to them.
-  
-Requires Imgur API key to be set as environmental varialbe `$IMGUR_API_KEY`. Usage:
+`vendor/lakion/mink-debug-extension/travis/tools/upload-screenshots "<glob_path>"` - uploades images to [imgur.com](http://imgur.com) and returns a list of file names and urls to them.
```
Comment (lchrusciel) [inline README.md]: _urls to them_ => _theirs url_ ?

Comment (pamil) [inline README.md]: :+1:

Comment (pjedrzejewski) [inline README.md]: `their urls`

File: README.md (hunk 1 above)
Comment (lchrusciel) [inline README.md]: :+1: I was just not sure about that.

**Pull request merged**